.gitattributes

```diff
@@ -1,7 +1,6 @@
 * text=auto
 
 /.github export-ignore
-/bin export-ignore
```

Comment:
Before this change I was getting:
```shell
    Skipped installation of bin bin/blade-icons-generate for package blade-ui-kit/blade-icons: file not found in package
```

Reply:
Ah, that snuck in actually. Thanks for letting me know.